content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -325,16 +343,16 @@ Since we had no news from you for more than 20 days, I'll close this ticket to a
 Thank you
 ```
 
-#### Issue validated
+### Issue validated
 
 ```
 Hello,
 
 I reproduce the issue with PrestaShop version x.x.x, I'll add this to the backlog so it can be fixed.
 
-Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project's own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you.
+Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 
 
-If you fix the issue on your end, please [contribute it back to the project]({{< devdocs "contribute/contribute-pull-requests/" >}}). Remember that the more people contribute, the better PrestaShop becomes for everyone.
+If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop-project.org/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.
```

Comment:
```suggestion
If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop-project.org/8/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.
```

although the `devdocs` helper should've worked 🤔

Reply:
Fixed